code/modules/projectiles/updated_projectiles/gun_attachables.dm

```diff
@@ -555,19 +555,16 @@ obj/item/attachable/attack_hand(var/mob/user as mob)
 	name = "\improper M37 wooden stock"
 	desc = "A non-standard heavy wooden stock for the M37 Shotgun. Less quick and more cumbersome than the standard issue stakeout, but reduces recoil and improves accuracy. Allegedly makes a pretty good club in a fight too.."
 	slot = "stock"
-	wield_delay_mod = WIELD_DELAY_NORMAL
+	wield_delay_mod = WIELD_DELAY_FAST
```

Comment:
This means no wield delay.

Reply:
Like I said, it's additive; it is in addition to the base delay.

Look at how the angled grip works.